lms/envs/common.py

```diff
@@ -45,7 +45,7 @@
 
 ################################### FEATURES ###################################
 # The display name of the platform to be used in templates/emails/etc.
-PLATFORM_NAME = "Your Platform Name Here"
+PLATFORM_NAME = _('edX')
```

Comment:
I mean within the context of my PR. So this includes:

 - `PLATFORM_NAME`
 - `ANALYTICS_DASHBOARD_NAME`
 - `STUDIO_NAME`
 - etc